src/Php80/Php80.php

```diff
@@ -88,17 +88,22 @@ public static function preg_last_error_msg(): string
         }
     }
 
-    public static function str_contains(string $haystack, string $needle): bool
+    public static function str_contains(string $haystack = null, string $needle = null): bool
     {
+        // Cast to string to avoid deprecation notices
+        if (null === $needle) {
```

Comment:
I think it can be simply casted to string.

Reply:
Changed, thank @IonBazan!